alf/data_structures.py

```diff
@@ -481,3 +481,15 @@ def make_experience(time_step: TimeStep, alg_step: AlgStep, state):
         "priority",
     ],
     default_value=())
+
+# some basic structures used by offline replay buffer
+# reconstruction
+Info = namedtuple("Info", [
+    'rl',
```

Comment:
need some explanation about the fields. I noticed that "reward" and "repr" are always `{}` when this structure is used. What are the purpose of them?

Reply:
Added more comments. Removed "reward" and "repr" as they are not essential here.